src/item_location.cpp

```diff
@@ -342,6 +372,16 @@ const item *item_location::operator->() const
     return ptr->what;
 }
 
+item_location::type item_location::where() const
+{
+    return ptr ? ptr->where() : type::invalid;
+}
+
+tripoint item_location::position() const
+{
+    return ptr ? ptr->position() : tripoint();
```

Comment:
Should be `tripoint_min` - just `tripoint` will be `(0, 0, 0)`, which is in reality bubble